docs/imagecustomizer/api/configuration/partition.md

```diff
@@ -69,3 +69,29 @@ Supported options:
   This flag is only supported on GPT formatted disks.
 
   For further details, see: https://en.wikipedia.org/wiki/BIOS_boot_partition
+
+- `home`: A `/home` partition.
+
+- `linux-generic`: A generic Linux partition.
+
+  This is the default value.
+
+- `root`: The `/` partition.
+
+- `root-verity`: The verity hash partition for `/`.
+
+- `srv`: The `/srv` partition.
+
+- `swap`: A swap partition.
+
+- `tmp`: The `/var/tmp` partition.
+
+- `usr`: The `/usr` partition.
+
+- `usr-verity`: The verity hash partition for `/usr`.
+
+- `var`: The `/var` partition.
+
+- `xbootldr`: The `/boot` partition.
+
+In addition, a UUID string may be specified.
```

Comment:
Also this ie the format should probably be in the first line, rather than at the end

Reply:
The UUID string is a special case, not the norm. Hence, I want it at the end.

That being said, I did reformat this so that UUID strings are part of the list. Hopefully, this makes it clearer,